src/symbol.js

```diff
@@ -44,8 +44,10 @@ const getExistingSymbols = (document: any) => {
 
     let symbolsPage = getSymbolsPage(document);
```

Comment:
IIRC, there is a document.getOrCreateSymbolPage() method used somewhere else already. We probably want to use that

Reply:
I did see it used, but wasn't sure if its implementation was exactly the same as what was happening here.

I kicked the can down the road, because I wasn't entirely sure how long we would be keeping this current Symbols API.

The next thing up on my list is investigating how to control manually rendering SymbolMasters rather then relying on `inject`.